src/Oceananigans.jl

```diff
@@ -143,18 +139,36 @@ import Base:
     push!
 
 "Boolean denoting whether AMGX.jl can be loaded on machine."
-const hasamgx = @static Sys.islinux() ? true : false
+# TODO: find a way to check whether the libraries for AMGX and NETCDF 
+# (libamgxsh and libnetcdf, respectively) are installed on the machine
+const hasamgx   = @static (Sys.islinux() && Sys.ARCH == :x86_64) ? true : false
+
+const hasnetcdf =
+    try
+        using NCDatasets
+        true
+    catch
+        false
+    end

```

Comment:
Since `try-catch` with `using NCDatasets` doesn't work, should we do this for now?

```suggestion
const hasnetcdf = @static (Sys.islinux() && Sys.ARCH == :x86_64) ? true : false
```